cli/aws_orbit/data/kubectl/fsx_driver/base/controller.yaml

```diff
@@ -0,0 +1,82 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: fsx-csi-controller
+  namespace: kube-system
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: fsx-csi-controller
+  template:
+    metadata:
+      labels:
+        app: fsx-csi-controller
+        orbit/node-type: ec2
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+        orbit/node-type: ec2
```

Comment:
if these containers don't have to be on each ec2, should we target them for the `orbit/usage`: reserved group as well as `orbit/node-type`?

Reply:
maybe they do.. I had some issues moving them to env.  I need to try one more time and if its not working , they probably need to be on every node.